content/broker/broker-transition/_index.en.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Altinn 3 Broker Transition Solution
+linktitle: Transition Solution
+description: Altinn 2 Broker Transition to Altinn 3 Broker architecture. 
+tags: [architecture, solution]
```

Comment:
consider the tags on all files to: [broker, guide, transition] or equivalent.

Reply:
done